Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -252,6 +252,19 @@ class TelemetryWrapper {
         GleanMetrics.Device.authentication.set(AppAuthenticator.canAuthenticateDeviceOwner())
     }
 
+    // Function for recording metrics that are better recorded when terminating the application
+    static func recordWillTerminatePreferenceMetrics() {
+
+        let delegate = UIApplication.shared.delegate as? AppDelegate
+        if let normalTabsQuantity = delegate?.tabManager.normalTabs.count {
```

Comment:
Wondering if a much cleaner approach would be to modify it as  `recordWillTerminatePreferenceMetrics(manager: TabManager)` and make it a part of `TelemetryWrapper` extension which will help us avoid directly grabbing tab manager from AppDelegate. 

What do you think?

Reply:
Yeah we can do that 👍